### PR TITLE
Minor ux changes and word adjustments

### PR DESCRIPTION
**Pull request by goodroot**
Merges minor UX and word adjustments into current PR.

### CODE REVIEW

File: templates/admin/authorize.php [hunk 1]
```diff
@@ -28,7 +28,17 @@
             <?php endif; ?>
 
             <div class="card">
-                <p><?= __("Please enter your Site Search API key in the field below and click 'Authorize' to get started."); ?></p>
+                <p>
+                  <strong><?= __('New to Site Search?') ?></strong> <br/>
+                  <?= __('Sign up for a free trial account and get your Site Search API Key: <b><a href="https://swiftype.com/users/sign-up" target="_new">Start 14 Day Trial</b></a>'); ?>
```
Comment (qhoxie) [inline templates/admin/authorize.php]: @mriley we should consider adding UTM tracking to these links

Comment (mriley) [reply templates/admin/authorize.php]: Good call. Can someone please edit the create account link to be the following: https://app.swiftype.com/signup?utm_channel=setup-admin&utm_source=wordpress-web

I will create my own PR to change the other account creation link that is in the README file.

Comment (mriley) [reply templates/admin/authorize.php]: Actually, if it's easy enough, can someone else please just change the link at the top of the README instructions to read: "1. [Sign up for a free account](https://app.swiftype.com/signup?utm_channel=readme-web&utm_source=wordpress-web) on the Swiftype website."

Comment (goodroot) [reply templates/admin/authorize.php]: @mriley You got it -- great idea. 

For general link audit: https://github.com/elastic/swiftype-pubs/issues/267.

**Pull request merged**